lib/adhearsion/translator/asterisk/component/stop_playback.rb

```diff
@@ -6,17 +6,28 @@ module Adhearsion
   module Translator
     class Asterisk
       module Component
-        module StopByRedirect
+        module StopPlayback
           def execute_command(command)
             return super unless command.is_a?(Adhearsion::Rayo::Component::Stop)
             if @complete
               command.response = Adhearsion::ProtocolError.new.setup 'component-already-stopped', "Component #{id} is already stopped", call_id, id
             else
-              stop_by_redirect Adhearsion::Event::Complete::Stop.new
+              stop_playback Adhearsion::Event::Complete::Stop.new
               command.response = true
             end
           end
 
+          def stop_playback(complete_reason)
+            if ami_client.version.to_s >= '2.0'
```

Comment:
If we're checking the *protocol* version, then using `Gem::Version` is most likely wrong.  Won't that simply report the `ruby_ami` package version, instead of the AMI protocol version?